tests/unit/challenges/test_views.py

```diff
@@ -2345,6 +2351,61 @@ def test_create_challenge_phase_when_user_is_not_part_of_host_team(self):
         self.assertEqual(response.data["detail"], expected["error"])
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
+    def test_challenge_phase_start_greater_than_challenge_start(self):
```

Comment:
also please rename `start` and `end` in function names to `start_date` `end_date`